django/db/backends/oracle/creation.py

```diff
@@ -119,6 +120,39 @@ def _switch_to_test_user(self, parameters):
             self.connection.settings_dict['USER'] = parameters['user']
         real_settings['PASSWORD'] = self.connection.settings_dict['PASSWORD'] = parameters['password']
 
+    def _clone_test_db(self, suffix, verbosity, keepdb):
+        parameters = self._get_cloned_db_params(suffix)
+        settings = self.connection.settings_dict
+        main_login = f"{settings['SAVED_USER']}/{settings['SAVED_PASSWORD']}@{settings['NAME']}"
```

Comment:
Do we accept f-strings now?

Reply:
We're only supporting 3.6 and above in master, so f-strings are acceptable.

Reply:
Yeah, but the last time I encountered it, we still avoided its use. I may have missed something.

Reply:
Ahh right, I'm not sure if there's an official policy on it but it's appearing in [a few PRs now](https://github.com/django/django/pull/13207/files#diff-b31c0b9609438fe2598d068a906814f7R132). If we need to switch to `.format` or `%s` we can do this later on, it should be easy enough.

Reply:
I found it a lot easier than having to use ```%s``` every time. If we have to switch, it'll be pretty simple

Reply:
So I've [opened another discussion re f-strings](https://groups.google.com/d/topic/django-developers/XNhpOHam0zE/discussion). 

But, whether they're allowed or not, I'm not quite convinced by the readability of this particular example... 😬